src/zeit/retresco/connection.py

```diff
@@ -47,19 +48,29 @@ def get_keywords(self, search_string):
                 entity['entity_name'], entity['entity_type'])
 
     def get_all_topicpages(self):
-        page = 0
+        start = 0
+        # XXX figure out row number that balances number of requests vs.
+        # work the TMS has to do per request.
+        rows = 100
         while True:
-            page += 1
-            # XXX figure out row number that balances number of requests vs.
-            # work the TMS has to do per request.
-            response = self._request(
-                'GET /topic-pages',
-                params={'q': '*', 'rows': 100, 'page': page})
-            if not response['docs']:
+            response = self.get_topicpages(start, rows)
+            start += rows
+            if not response:
                 break
             for row in response['docs']:
                 yield row
```

Comment:
Funktioniert `response['doc']` hier noch? Das wird doch jetzt in `get_topicpages` schon gemacht und sollte hier bereits eine Liste/`Result` sein.